app/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
         minSdkVersion 15
         targetSdkVersion 23
         versionCode 80
-        versionName "0.8.0"
+        versionName "0.9.0"
```

Comment:
This is not be clubbed with any other change. 
Version bumps are NOT dependent on libs, but on the features we add to the app.